core/include/kernel/msg_param.h

```diff
@@ -45,6 +45,19 @@ enum msg_param_mem_dir {
 };
 
 /**
+ * msg_param_mobj_from_noncontig_param() - construct mobj from non-contigous
+ * list of pages.
+ *
+ * @param - pointer to msg_param with OPTEE_MSG_ATTR_NONCONTIG flag set
+ * @bool  - true if buffer needs to be mapped into OP-TEE address space
+ *
+ * return:
+ *	mobj or NULL on error
+ */
+struct mobj *msg_param_mobj_from_noncontig_param(
```

Comment:
`msg_param_mobj_from_noncontig()` is long enough for my taste at least.

When parameters end up too far right to line up comfortable with the opening `(` please format as:
```
struct mobj *
msg_param_mobj_from_noncontig_param(const struct optee_msg_param *param,
                                    bool map_buffer);
```

This seems to be the preferred style during the circumstances in the Linux kernel too.